portal/models/login.py

```diff
@@ -43,17 +43,22 @@ def login_user(user, auth_method=None):
         # log user back out, in case a flow already promoted them
         flask_user_logout()
 
-        token = user.generate_otp()
+        code = user.generate_otp()
         session['user_needing_2fa'] = user.id
         session['pending_auth_method'] = auth_method
-        current_app.logger.debug(f"OTP for {user.id}: {token}")
+        current_app.logger.debug(f"2FA OTP for {user.id}: {code}")
         email = EmailMessage(
-            subject=_("TrueNTH Access Token"),
-            body=_("One Time Authentication Token: %s" % token),
+            subject=_("TrueNTH Access Code"),
+            body=_("One Time Authentication Code: %s" % code),
```

Comment:
I don't think this is lazily evaluated, meaning babel won't be able to extract it...

```suggestion
            body=_("One Time Authentication Code: %(code)s", code=code),
```